src/sass/components/_button.scss

```diff
@@ -29,6 +34,16 @@
   font-size: 13px;
   line-height: 22px;
 
+  &.active {
+    color: #d58e32;
```

Comment:
może warto wydzielić kolory do osobnego pliku i ustawić globalne zmienne? To pozwoliłoby na używanie kolorów bez potrzebny pamiętania HEX